src/Controller/ActionHandler.py

```diff
@@ -0,0 +1,255 @@
+import sys
+
+from PyQt5 import QtGui, QtWidgets
+
+from src.Model.CalculateImages import get_pixmaps
+from src.Model.PatientDictContainer import PatientDictContainer
+
+
+class ActionHandler:
```

Comment:
so what is this, why is it here, and how is it used?
While there are single comment lines below that say "X action" followed by code that is
self.zzz_X_action, there is no documentation that helps someone understand the bigger picture.

https://www.pythonforbeginners.com/basics/python-docstrings
or any equivalent introduction to docstrings (that's the first one that came up when I put
docstring python in the duckduckgo search box)